core/src/main/java/hudson/model/Computer.java

```diff
@@ -507,9 +507,11 @@ public final long getConnectTime() {
 
     /**
      * CLI command to disconnects this node.
+     *
+     * @param cause
+     *      Record the note about why you are disconnecting this node
      */
-    @CLIMethod(name="disconnect-node")
-    public void cliDisconnect(@Option(name="-m",usage="Record the note about why you are disconnecting this node") String cause) throws ExecutionException, InterruptedException {
+    public void cliDisconnect(String cause) throws ExecutionException, InterruptedException {
```

Comment:
@pjanouse Many of them, AFAICT, were just reused regular API. The name `cliDisconnect` indicates that's not what this is supposed to be.